css/de_theme.css

```diff
@@ -9333,59 +9365,6 @@ ul.privatemsg-message-actions {
   background-color: #704a64 !important;
 }
 
-.view-project-featured-object {
```

Comment:
@nikathone , @jefferya and I are wondering why you deleted this portion that is governing the behaviour of the featured object on project pages? is there a connection to the the sponsors section? Just for clarity, we would like to keep both the feature object and the sponsors sections.

Reply:
See more details about this at https://github.com/cwrc/de_theme/commit/12b0919ebfa11eb77c63328cb45a6a66852170bc#commitcomment-29284893